### PR TITLE
Hotels: Update PricePopup with new design

### PR DESCRIPTION
**Pull request by tbergq**
Shared: Add type info and circled prop to AdaptableBadge
Shared: Update slider to match with new design

### CODE REVIEW

File: app/hotels/src/filter/price/PricePopup.js [hunk 1]
```diff
@@ -77,30 +78,35 @@ export default class PricePopup extends React.Component<Props, State> {
...
-              <Price amount={start * daysOfStay} currency={currency} />
-            }
-            startValue={start}
-            endLabel={<Price amount={end * daysOfStay} currency={currency} />}
-            endValue={end}
-          />
+            <AdaptableBadge
+              translation={
+                <TranslationFragment>
+                  <Price amount={start * daysOfStay} currency={currency} />
```
Comment (RobinCsl) [inline app/hotels/src/filter/price/PricePopup.js]: Should we use the number library to compute the price?

File: packages/shared/src/AdaptableBadge.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// @flow strict
+// @flow
```
Comment (RobinCsl) [inline packages/shared/src/AdaptableBadge.js]: Why this change?

File: packages/shared/src/AdaptableBadge.js [hunk 3]
```diff
@@ -51,4 +60,10 @@ const style = StyleSheet.create({
   text: {
     color: defaultTokens.paletteWhite,
   },
+  info: {
+    backgroundColor: defaultTokens.paletteBlueNormal,
+  },
+  circled: {
+    borderRadius: 11,
```
Comment (RobinCsl) [inline packages/shared/src/AdaptableBadge.js]: The designer is not using Orbit values?

File: packages/shared/src/forms/SliderMarker.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+import StyleSheet from '../PlatformStyleSheet';
+
+export default function SliderMarker() {
```
Comment (RobinCsl) [inline packages/shared/src/forms/SliderMarker.js]: There is no "pressed" state?

File: app/hotels/src/filter/price/PricePopup.js (hunk 1 above)
Comment (tbergq) [reply app/hotels/src/filter/price/PricePopup.js]: You mean decimal? No, since this particular value still comes as Float from the server.

File: packages/shared/src/AdaptableBadge.js (hunk 2 above)
Comment (tbergq) [reply packages/shared/src/AdaptableBadge.js]: It had it's reason, then I improved it, so yes, I will add it back tomorrow. Thanks 😊

File: packages/shared/src/AdaptableBadge.js (hunk 3 above)
Comment (tbergq) [reply packages/shared/src/AdaptableBadge.js]: I think you already know the answer to that 😞

File: packages/shared/src/forms/SliderMarker.js (hunk 4 above)
Comment (tbergq) [reply packages/shared/src/forms/SliderMarker.js]: Should it be?

**Comment by tbergq** [comment]
@RobinCsl 🚀

File: packages/shared/src/forms/SliderMarker.js (hunk 4 above)
Comment (RobinCsl) [reply packages/shared/src/forms/SliderMarker.js]: As a user, I would like some visual feedback that I pressed the handle. But I don't know if the designer has this in mind? I could not find anything in Figma so 🤷‍♂

Comment (tbergq) [reply packages/shared/src/forms/SliderMarker.js]: When you have your finger over the marker, you don't see it

Comment (RobinCsl) [reply packages/shared/src/forms/SliderMarker.js]: My finger is not so big :D I guess that's because I'm using the mouse with the emulator.

Comment (tbergq) [reply packages/shared/src/forms/SliderMarker.js]: > I guess that's because I'm using the mouse with the emulator

🙈 

I can't see the marker when I put my finger on top of it, but when dragging, it becomes partially visible, but by then, I already know that I have activated it
